setup.py

```diff
@@ -1,21 +1,21 @@
-import os
-
+"""A setuptools-based script for installing FauxFactory."""
 from distutils.core import setup
```

Comment:
We should use setuptools instead of distutils. You can see an example of what the resultant `setpy.py` file looks like [here](https://github.com/SatelliteQE/nailgun/blob/master/setup.py). You can see the reasoning [here](https://github.com/SatelliteQE/nailgun/blob/master/setup.py) (ref 5).